### ISSUE TITLE
doesn't work on windows unless using cygwin, and with a patch

### ISSUE THREAD

**Issue by mikesmullin** [mikesmullin/borg#7]
there is a subtle issue with how to resolve symlinks.

probably need to duplicate the `./bin/borg` file into two, including: `./bin/borg.bat` for Windows-Cygwin, and the original for Mac/Linux systems.

see also:
https://github.com/mikesmullin/borg/commit/c338877e5ed8eb3dc64931abf9eaf7349734d328

**Comment by kewubenduben** [comment]
just wondering if borg happen to fully support windows. Considering there's some implementation, how'd you test it and until where does it support so far?

**Comment by sbimikesmullin** [comment]
no, it does not fully support windows. i was using it successfully with Cygwin for a while on my Windows 8 Surface Pro 3. but i stopped when i ran into quirks of Cygwin (some limit on the max. number of processes of node that can run; would trigger random errors. also happens with git.)  to get it to support Windows without Cygwin would probably be not very fun. But I'd approach it like Gradle does with its gradlew wrapper.
